core-library/src/main/java/org/silverpeas/core/admin/service/GroupSynchronizationRule.java

```diff
@@ -25,6 +25,7 @@
 
 import com.novell.ldap.LDAPException;
 import com.novell.ldap.LDAPLocalException;
+import org.silverpeas.core.SilverpeasExceptionMessages;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'org.silverpeas.core.SilverpeasExceptionMessages'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)